src/alto-ui/Tag/Tag.scss

```diff
@@ -0,0 +1,48 @@
+@import '../scss/inc';
+
+.Tag {
+  background-color: $blue-10;
+  border: 1px solid $blue-20;
```

Comment:
```suggestion
  border: $border-width-thin solid $blue-20;
```

Reply:
good catch but we are not using those variables, I need to delete them.